tests/library/modernizr.spec.ts

```diff
@@ -57,7 +57,7 @@ it('Safari Desktop', async ({ browser, browserName, platform, server, headless }
     expected.publickeycredential = false;
     expected.mediastream = false;
     if (headless)
-      expected.todataurljpeg = false;
+      expected.todataurlwebp = true;
```

Comment:
This seems to be related to https://github.com/microsoft/playwright/issues/29097 - looks like we can close it now?

Reply:
Yes, it does fix it.